app/charges/charges_response.py

```diff
@@ -8,6 +8,8 @@
 class Response():
 	AddSuccess = {'success': 'Charge successfully created.'}
 	AddError = {'error': 'Charge couldnt be created, check data.'}
-	UsrCommDontExist = {'error': 'User or committee do not exist.'}
+	UsrChargeDontExist = {'error': 'User or charge does not exist.'}
```

Comment:
Is there any reason why these are both in one? Why not have a ChargeDoesNotExist and UserDoesNotExist?

Reply:
this is consistent with how we do messages everywhere else